src/bp/ui-admin/src/Pages/Workspace/Bots/index.tsx

```diff
@@ -70,6 +73,7 @@ class Bots extends Component<Props> {
   componentDidMount() {
     this.props.fetchBots()
     this.props.fetchBotHealth()
+    this.props.profile && this.props.profile.isSuperAdmin && this.props.fetchModules()
```

Comment:
Can you check that modules have not been fetched already. Juste to make sure we dont fetch them everytime ee mount that component

Reply:
:+1: